config/manager/manager.yaml

```
@@ -161,7 +161,7 @@ spec:
             secretKeyRef:
               key: ceph-username
               name: rook-ceph-mon
-        image: quay.io/cybozu/ceph:18.2.1.1
+        image: quay.io/cybozu/ceph:v19.2.1
```

This image "quay.io/cybozu/ceph:v19.2.1" doesn't exists.

```suggestion
        image: quay.io/ceph/ceph:v19.2.1
```

NOTE: Since this manifest is not used for now, I'm considering to remove it later.